spinalcordtoolbox/utils.py

```diff
@@ -430,6 +453,46 @@ def sct_progress_bar(*args, **kwargs):
     return tqdm.tqdm(*args, **kwargs)
 
 
+def zipdir(path_to_zip, fname_zip='archive.zip'):
+    """
+    Create zip archive.
+    :param path_to_zip: file or directory
+    :param fname_zip: name of output zip archive
+    :return: path to zip archive
+    """
+    import zipfile
+    import os
+
+    fname_zip = abspath(fname_zip)
+    currentDir = os.getcwd()
+    try:
+        relroot = abspath(path_to_zip)
+        os.chdir(path_to_zip)
```

Comment:
There is a better way than using the dreaded `chdir()`

Reply:
fixed in 0407fc2fc5b2782080d62e801e42d94c76996279